script.tag-generator/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.tag-generator"
+       name="Tag Generator"
+       version="0.5.1"
```

Comment:
@wellspokenman please be aware that this changes your versioning system in a "big" way. Let me know it it's okay. You might also want to do this to your github repo then.